### PR TITLE
Make unit tests working with PHP 5.3

### PR DESCRIPTION
**Pull request by Al2Klimov**

### CODE REVIEW

**Comment by Al2Klimov** [comment]
@lippserd: I don't get why even this doesn't work. Why don't we just drop PHP 5.3 as in other modules?

**Comment by Thomas-Gelf** [comment]
@Al2Klimov: Travis changed base OS and broke the tests. @lazyfrosch fixed this for other modules and should be able to help you to fix this.

Regarding PHP 5.3: the current module supports Icinga Web 2.4.1+ and PHP 5.3. We might be able to raise the requirements with the next feature release, but not with a patch release shipping some minor fixes.

**Comment by lazyfrosch** [comment]
I guess this errors because `env` is not merged.

It should work as a hash on global level, and on the matrix level.

```yaml
env:
  ICINGAWEB_VERSION: 2.5.0
```

Also have a look at [setup_vendor.sh](https://github.com/Icinga/icingaweb2-module-director/blob/master/test/setup_vendor.sh) to remove all the clutter from the yaml file.

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
 
+matrix:
+  include:
+  - php: '5.3'
+    dist: precise
+    sudo: required
+    env:
+    - PHPCS_VERSION=2.9.1
+    - LOCALE_GEN=1
+    - ENABLE_LDAP=1
```
Comment (lazyfrosch) [inline .travis.yml]: ~~Not needed here.~~ Only `PHPCS_VERSION` is needed here...

**Comment by lazyfrosch** [comment]
@Al2Klimov for now, let's switch to `setup_vendor.sh` and set the default versions there.

`PHPCS_VERSION` needs to be pinned down for php 5.3

**Comment by lazyfrosch** [comment]
Still missing:
* Latest IW2 Version
* Latest PHPCS version

**Comment by lippserd** [comment]
@lazyfrosch Yes but this should be done in a separate PR.

**Pull request merged**